mockgcp/mockcompute/regionalforwardingrulev1.go

```diff
@@ -82,6 +82,9 @@ func (s *RegionalForwardingRulesV1) Insert(ctx context.Context, req *pb.InsertFo
 	if obj.NetworkTier == nil {
 		obj.NetworkTier = PtrTo("PREMIUM")
 	}
+	if *obj.LoadBalancingScheme == "" {
+		obj.LoadBalancingScheme = nil
+	}
 
```

Comment:
can I ask why we need this for the api bundle?

Reply:
PSC forwarding rule requires LoadBalancingScheme to be set to `""`. If it's `""`, then it should not appear in the response body. So I add this in order to to match the realGCP log.